### DISCUSSION TITLE
Welcome to homebox Discussions!

### DISCUSSION THREAD

**Discussion by progmaticltd** [progmaticltd/homebox#366] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arodier** [comment]
I am actually rewriting the documentation, to make it more focused on the installation itself.
For the features list, I decided it would be a better idea to put everything on a dedicated website: https://homebox.space/
It is a work in progress, but the source code is available, and as usual, contributions are welcome.
